stake-pool/js/src/index.ts

```diff
@@ -90,6 +101,32 @@ export async function getStakePoolAccount(
   };
 }
 
+/**
+ * Retrieves and deserializes a Stake account using a web3js connection and the stake address.
+ * @param connection: An active web3js connection.
+ * @param stakeAccount: The public key (address) of the stake account.
+ */
+export async function getStakeAccount(
+  connection: Connection,
+  stakeAccount: PublicKey,
+): Promise<StakeAccount> {
+  const account = await connection.getAccountInfo(stakeAccount);
+
+  if (!account) {
+    throw new Error('Invalid stake account');
+  }
+
+  return {
+    pubkey: stakeAccount,
+    account: {
+      data: AccountLayout.decode(account.data),
```

Comment:
Be careful. this is trying to decode an SPL Token Account, and *not* a stake account.  You'll need to use the stake account layout to decode this properly